src/pages/ProductList/index.js

```diff
@@ -16,17 +16,34 @@ import { ProductsWrap } from './styled'
 
 const Products = ({ match, location, addProduct }) => {
   const { page } = qs.parse(location.search.substr(1))
+  const [count, setCount] = useState(25)
 
   const { data: res, isLoading } = useApi(
-    () => getProducts({ page: { number: page } }),
+    () => getProducts({ page: { number: page, size: count } }),
     [page]
   )
 
+  useEffect(() => {
+    // TODO: Rerender here
```

Comment:
`useApi` hook already leverages `useEffect` hook inside of it, so you don't need to add another `useEffect`. You can pass `resolveCondition` argument as a second argument to `useApi` hook

Reply:
got it, all fixed!